toptal/picasso
# style(Sidebar): [FX-1056] apply v5 design

[FX-1056](https://toptal-core.atlassian.net/browse/FX-1056)

### Description

[Abstract](https://share.goabstract.com/edfd5dd8-2dec-40ce-bc3b-d4a132a1851d)
[Picasso](https://picasso.toptal.net/?path=/story/components-folder--sidebar#default)

**Changes**

- `max-width: 236px`

  - If a sidebar item doesn’t fit within the set max width the overflow should be handle with an ellipsis (should already work by default)

- Background and hover colors

- Updated collapse/expand icons

- Spacing

  - `1rem` spacing between `Sidebar` and `Sidebar.Item`

  - `1rem` padding in `Siderbar.Item` on all sides

**1 to 1 scale Abstract screenshots (click to expand)**

<img width="1190" alt="Screenshot 2020-09-01 at 18 27 58" src="https://user-images.githubusercontent.com/1390758/91839045-fe060d80-ec80-11ea-85e4-37534892a800.png">

---

<img width="269" alt="Screenshot 2020-09-01 at 18 28 11" src="https://user-images.githubusercontent.com/1390758/91839039-fba3b380-ec80-11ea-85b0-a15cb4e13b24.png">


**Additional information**

- the https://toptal-core.atlassian.net/browse/FX-1170 was discovered;
- 📚 prepared according to the https://github.com/toptal/picasso/pull/1474.

### How to test

- navigate to the temploy and make sure that applied style corresponds to the Abstract one;
- use the LastPass account at https://browserstack.com/ to verify that changes work in the latest supported browsers;
- please see the https://jenkins-build.toptal.net/job/picasso-pr-specs/3470//Visual_20Regression_20Tests/ for visual differences.

### Screenshots

![demo](https://user-images.githubusercontent.com/1390758/91839267-5d641d80-ec81-11ea-9bd1-1efb5bee4245.gif)

### Review


<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run all` - Run  whole pipeline
- `@toptal-bot run danger` - Danger checks
- `@toptal-bot run lint` - Run linter
- `@toptal-bot run test` - Run jest
- `@toptal-bot run build` - Check build
- `@toptal-bot run test:visual` or `@toptal-bot run visual` - Run visual tests
- `@toptal-bot run deploy:documentation` - Deploy documentation


</details>
